benches/pk_encryption/ring_lpr.rs

```diff
@@ -0,0 +1,288 @@
+
+    /// This was previously generated by [`RingLPR::gen`]
+    /// with the parameters set in [`RingLPR::default`].
+    const SMALL_PK0: &str =
+        "{\"poly\":\"16  2328 637 2200 149 2177 1070 1412 1865 2321 1608 2340 2145 1431 740 368 556\"}";
+    const SMALL_PK1: &str =
+        "{\"poly\":\"16  480 1934 2216 296 3 1576 2255 1137 1631 2167 1322 1823 93 15 1749 1976\"}";
+    const SMALL_SK: &str = "{\"poly\":\"16  2395 3 2396 0 2398 8 2397 2 2396 4 3 0 1 5 3 3\"}";
+    const SMALL_CIPHER0: &str =
+        "{\"poly\":\"16  1689 12 1037 381 1212 847 220 2169 2006 739 713 1329 1585 290 891 1972\"}";
+    const SMALL_CIPHER1: &str =
+        "{\"poly\":\"16  120 274 282 1258 1404 581 1969 2206 1747 1413 1179 259 1367 2394 2275 1253\"}";
+
+    /// Loads one of the above precomputed public key suitable to:
+    /// - [`SizeN::Small`]: [`RingLPR::default`]
+    /// - [`SizeN::Large`]: [`RingLPR::secure128`]
+    pub(super) fn load_precomputed_pk(size_n: &SizeN) -> (PolynomialRingZq, PolynomialRingZq) {
+        if size_n == &SizeN::Large {
+            let modulus =
+                qfall_crypto::utils::common_moduli::new_anticyclic(512, 92897729).unwrap();
+
+            let poly0: PolyOverZ = serde_json::from_str(LARGE_PK0).unwrap();
+            let poly1: PolyOverZ = serde_json::from_str(LARGE_PK1).unwrap();
+
+            let pk0 = PolynomialRingZq::from((poly0, &modulus));
+            let pk1 = PolynomialRingZq::from((poly1, &modulus));
+
+            (pk0, pk1)
+        } else {
+            let modulus = qfall_crypto::utils::common_moduli::new_anticyclic(16, 2399).unwrap();
+
+            let poly0: PolyOverZ = serde_json::from_str(SMALL_PK0).unwrap();
+            let poly1: PolyOverZ = serde_json::from_str(SMALL_PK1).unwrap();
+
+            let pk0 = PolynomialRingZq::from((poly0, &modulus));
+            let pk1 = PolynomialRingZq::from((poly1, &modulus));
+
+            (pk0, pk1)
+        }
+    }
+
+    /// Loads one of the above precomputed secret key suitable to:
+    /// - [`SizeN::Small`]: [`RingLPR::default`]
+    /// - [`SizeN::Large`]: [`RingLPR::secure128`]
+    pub(super) fn load_precomputed_sk(size_n: &SizeN) -> PolynomialRingZq {
+        if size_n == &SizeN::Large {
+            let modulus =
+                qfall_crypto::utils::common_moduli::new_anticyclic(512, 92897729).unwrap();
+            let poly: PolyOverZ = serde_json::from_str(LARGE_SK).unwrap();
+            let sk = PolynomialRingZq::from((poly, &modulus));
+
+            sk
+        } else {
+            let modulus = qfall_crypto::utils::common_moduli::new_anticyclic(16, 2399).unwrap();
+            let poly: PolyOverZ = serde_json::from_str(SMALL_SK).unwrap();
+            let sk = PolynomialRingZq::from((poly, &modulus));
+
+            sk
+        }
```

Comment:
Here was also a clippy warning
```suggestion
        match size_n {
            SizeN::Large => {
                let modulus =
                    qfall_crypto::utils::common_moduli::new_anticyclic(512, 92897729).unwrap();
                let poly: PolyOverZ = serde_json::from_str(LARGE_SK).unwrap();
                PolynomialRingZq::from((poly, &modulus))
            }
            SizeN::Small => {
                let modulus = qfall_crypto::utils::common_moduli::new_anticyclic(16, 2399).unwrap();
                let poly: PolyOverZ = serde_json::from_str(SMALL_SK).unwrap();
                PolynomialRingZq::from((poly, &modulus))
            }
        }
```